data/noiseprofiles.json

```diff
@@ -6064,7 +6064,7 @@
       ]
     },
     {
-      "maker": "Ricoh",
```

Comment:
This breaks all other Ricoh cameras

Reply:
I don't think so, there is only GR besides this new GR II. As far as I know the maker entry is the same for both cameras. EXIF say "RICOH IMAGING COMPANY, LTD."
Are you sure the GR was ever tested?

Reply:
In case GR need the simple "Ricoh" header, I will have to add above as a separate entry, is that acceptable?